red-app/src/main/resources/static/js/candidate.js

```diff
@@ -0,0 +1,68 @@
+//***************************************************************
+//  CANDIDATE
+//***************************************************************
+
+ var candidateLanguage;
+ $(document).ready(function() {
+ $("#LanguageButtonPL").click(function (){
+ candidateLanguage = $("#LanguageButtonPL").attr("data-arg");
+   $.get( "/jobTitles/language/" + candidateLanguage, function( data ) {
+
+   var list = document.getElementById("resultJobTitle");
+   while (list.hasChildNodes()) {
```

Comment:
Jeśli tu jest usuwanie wszystkich dzieci to zrób to JQuery $("#resultJobTitle").empty();